src/store/middlewares/savedAccounts.js

```diff
@@ -8,26 +8,40 @@ import { getLastActiveAccount } from '../../utils/savedAccounts';
 import getNetwork from '../../utils/getNetwork';
 import networks from '../../constants/networks';
 
+import { getAutoLogInData, shouldAutoLogIn } from '../../utils/login';
+import settings from '../../constants/settings';
+
 const savedAccountsMiddleware = (store) => {
   setImmediate(() => {
     const accountsRetrievedAction = accountsRetrieved();
-    const savedAccounts = accountsRetrievedAction.data;
+    // const savedAccounts = accountsRetrievedAction.data;
     store.dispatch(accountsRetrievedAction);
 
-    if (savedAccounts && savedAccounts.lastActive) {
-      const account = savedAccounts.lastActive;
-      const network = Object.assign({}, getNetwork(account.network));
-
-      /* istanbul ignore if  */
-      if (account.network === networks.customNode.code) {
-        network.address = account.peerAddress;
-      }
+    const autologinData = getAutoLogInData();
+    const passphrase = autologinData[settings.keys.autologinKey];
+    const network = { ...networks.customNode, address: autologinData[settings.keys.autologinUrl] };
```

Comment:
Please use setImmediate in different middleware -- I will be removing all savedAccounts code next week